### PR TITLE
Fix recommendations

### PR DESCRIPTION
**Pull request by Shadowghost**
Didn't find any issue describing this but there were reports on Matrix.

### CODE REVIEW

**Review by Bond-009 — APPROVED** [review]

**Review by 1337joe — APPROVED** [review]

File: Emby.Server.Implementations/Data/SqliteItemRepository.cs [hunk 1]
```diff
@@ -2403,7 +2403,7 @@ private void SetFinalColumnsToSelect(InternalItemsQuery query, List<string> colu
                 }
 
                 // genres, tags, studios, person, year?
-                builder.Append("+ (Select count(1) * 10 from ItemValues where ItemId=Guid and CleanValue in (select CleanValue from itemvalues where ItemId=@SimilarItemId))");
+                builder.Append("+ (Select count(1) * 10 from ItemValues where ItemId=Guid and CleanValue in (select CleanValue from ItemValues where ItemId=@SimilarItemId))");
```
Comment (cvium) [inline Emby.Server.Implementations/Data/SqliteItemRepository.cs:2406]: Unnecessary change, no? I'm pretty sure it's case insensitive

Comment (Shadowghost) [reply Emby.Server.Implementations/Data/SqliteItemRepository.cs:2406]: Me too, but we did it case sensitive for all other tables except this one, so for consitencie's sake I changed it. I cna revert that if required.
